xled/control.py

```diff
@@ -197,7 +199,8 @@ def get_mode(self):
         url = urljoin(self.base_url, "led/mode")
         response = self.session.get(url)
         app_response = ApplicationResponse(response)
-        assert sorted(app_response.keys()) == [u"code", u"mode"]
+        required_keys = [u"code", u"mode", u"shop_mode"]
```

Comment:
Is `shop_mode` required or not? In the previous commit it was removed and here it was put back.

Reply:
It caused tests to fail, because it wasn't in the cassette for that test. It's not required for library operation so I took it back out instead of adding it to the cassette.

Reply:
In any case, I dropped it from the commit it was introduced, re-ordered, and squashed everything so it's like it was never there.